source/documentation/gds_on_site_backups.md

```diff
@@ -0,0 +1,23 @@
+# On site MYSQL backups
+
+All the GovWifi databases are backed up at 3am daily by the internal GDS build server.
+In order to connect to mysql, it has to go through the bastion server.
+
+The script that does the backups is located in: 
+
+`/root/Scripts/govwifi-backup.sh`
+
+You can ssh into this server:
+
+`ssh -i ~/.ssh/bob -vvvv bob@ah-govwf-d-01.dmz.gds`
+
+You need to be on a VPN to gain access to this server.
+Please speak to the security team to get you set up on this.
```

Comment:
I feel like we may need to onboard individuals to that server. I'm not certain if this responsibility lies with IT or the team. Worth clarifying?

Reply:
Sorry, what I meant is that VPN is not enough. There needs to be the `useradd -m -G sudo USERNAME` action performed by someone with existing sudo access.